platform/ios/benchmark/locations.cpp

```diff
@@ -4,15 +4,10 @@ namespace mbgl {
 namespace bench {
 
 const std::vector<Location> locations = {
-    { "paris",             2.3411,   48.8664, 11,   0.0 },
-    { "paris2",            2.3516,   48.8356, 13, 273.8 },
-    { "alps",             10.6107,   46.9599,  6,   0.0 },
-    { "us east",         -84.3395,   36.9400,  5,   0.0 },
-    { "greater la",     -117.9529,   34.0259,  9,   0.0 },
-    { "sf",             -122.4202,   37.7625, 14,   0.0 },
-    { "oakland",        -122.2328,   37.8267, 12,   0.0 },
-    { "germany",           9.2280,   50.9262,  6,   0.0 },
+    {"paris", 2.3411, 48.8664, 11, 0.0},        {"paris2", 2.3516, 48.8356, 13, 273.8},
```

Comment:
This code uses fewer lines, but it’s a bit less readable than the manually tabulated struct we had been using. Not the end of the world for this particular case, but worth pointing out in case there are more severe cases.